Library/Homebrew/dependency_collector.rb

```diff
@@ -81,6 +81,10 @@ def bzip2_dep_if_needed(tags)
     Dependency.new("bzip2", tags) unless which("bzip2")
   end
 
+  def gpatch_dep_if_needed(tags)
```

Comment:
```suggestion
  def self.gpatch_dep_if_needed(tags)
```
`def self.` makes it a class method rather than an instance method.

Move this function below `java_dep_if_needed`.